packages/dx-react-grid-material-ui/src/templates/table.jsx

```diff
@@ -3,6 +3,9 @@ import * as PropTypes from 'prop-types';
 import classNames from 'classnames';
 import TableMUI from '@material-ui/core/Table';
 import { withStyles } from '@material-ui/core/styles';
+// eslint-disable-next-line no-restricted-imports
+import { RootRef } from '@material-ui/core';
```

Comment:
Why don't we use the [recommended](https://material-ui.com/guides/minimizing-bundle-size/#how-to-reduce-the-bundle-size) import statement `import RootRef from '@material-ui/core/RootRef';`?